content/jobs/README.md

```diff
@@ -0,0 +1,46 @@
+# Job Post Instructions
+
+---
+
+Hey there! If you're a local-ish tech company looking to hire talent we at NewHaven.io would love to help out by putting some information about your organization and open role(s) on our website.
+
+Just clone this repository, add your information to [this page](https://github.com/newhavenio/newhavenio/tree/master/content/jobs/jobs.yaml), and submit a pull request.
```

Comment:
Maybe link to the GH clone url or a doc on how to do that

Reply:
Added a URL to the GitHub cloning docs.